storage/statedb/hasher.go

```diff
@@ -132,7 +133,7 @@ func (h *hasher) hashRoot(n node, db *Database, force bool) (node, node) {
 	// Trie not processed yet or needs storage, walk the children
 	collapsed, cached := h.hashChildrenFromRoot(n, db)
 	hashed, lenEncoded := h.store(collapsed, db, force)
-	// Cache the hash of the node for later reuse and remove
+	// TrieNodeCache the hash of the node for later reuse and remove
```

Comment:
```suggestion
	// Cache the hash of the node for later reuse and remove
```

Reply:
Thanks. I changed this one and another mistake.